zeus/monitor/power.py

```diff
@@ -239,27 +242,25 @@ def get_power(self, time: float | None = None) -> dict[int, float] | None:
 
 
 def _polling_process(
-    nvml_gpu_indices: list[int],
+    gpu_indices: list[int],
```

Comment:
Just checking my understanding with you -- Since the `NVIDIAGPUs` class will take of the NVML-CUDA device index mapping, now everything can just forget about NVML indices and all operate on CUDA indices, right?